rust/osm2lanes/src/lib.rs

```diff
@@ -153,7 +153,8 @@ mod tests {
             let actual = get_lane_specs_ltr(test.tags.clone(), &cfg);
             if actual != test.output {
                 ok = false;
-                println!("For input (example from {}):", test.way);
+                println!("For input (example from https://www.openstreetmap.org/way/{}):",
```

Comment:
Please take a look, because I'm not sure about Rust code style and how line longer than 100 characters should be formatted.

Reply:
Ah good point. If you have Rust and the cargo toolchain installed locally, you can just do `cargo fmt` and it'll fix it. If you don't have it installed, feel free to leave it this way, and I'll run it when I do the next bit of work

Reply:
Surprisingly it was easy to install and configure. I'll use it, thanks.

Reply:
Yeah, the Rust build tools are super straightforward. Everything looks good to me!